packages/base-map/src/index.js

```diff
@@ -74,10 +74,25 @@ L.Evented.include({
 class BaseMap extends Component {
   overlays = [];
 
+  constructor(props) {
+    super(props);
+    const { baseLayers } = props;
+    if (baseLayers && baseLayers.length > 0) {
+      this.state = this.getMapboxWordmarkUpdate(baseLayers[0].url);
```

Comment:
If we're adding state to this component, it seems like the more appropriate/extensible way to handle would be to
- initialize the `layerIndex` to zero
- set `LayersControl.BaseLayer#checked` below to `i === this.state.layerIndex`
- remove the `baselayerchange` listener in `componentDidMount` in favor of adding the `onBaseLayerChange` prop to the `<Map>` component
-  add a `handleBaseLayerChange` class method that updates internal state layerIndex.

This is a lot to describe... so I just made a PR: https://github.com/ibi-group/otp-ui/pull/4.